octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -921,6 +921,18 @@ def serve_pictures(
                 saveNext = True
                 latest = cam.lastPic()
             else:
+                if cam.camera_error:
+                    self.stopEvent.set()
+                    self._logger.error(
+                        "Camera error: %s, disabling camera", cam.camera_error
+                    )
+                    self._plugin.user_notification_system.show_notifications(
+                        self._plugin.user_notification_system.get_notification(
+                            "err_cam_conn_err",
+                            err_msg=[str("camera error")],
+                            err_code="E-01FF-1006",
```

Comment:
I think there was a module for the error codes, no?

Reply:
you are right :)